script/bootstrap.py

```diff
@@ -1,5 +1,10 @@
 #!/usr/bin/env python
```

Comment:
actually I believe this should be `#!/usr/bin/env vpython3` now

Reply:
updated, thanks